CONTRIBUTING.md

```diff
@@ -39,23 +39,47 @@ contribute is to learn how to use it and put your changes on a Git repository.
 There's a plenty of documentation about Git -- you can start with the [Pro Git
 book](http://git-scm.com/book/).
 
-### Forks + GitHub Pull requests
+
+### Setting up a Development Environment
+
+To set up your local development environment for contributing to the main
+repository [nltk/nltk](https://github.com/nltk/nltk/):
+
+- Fork the [nltk/nltk](https://github.com/nltk/nltk/) repository on GitHub
+  to your account;
+- Clone your forked repository locally
+  (`git clone https://github.com/<your-github-username>/nltk.git`);
+- Run `cd nltk` to get to the root directory of the `nltk` code base;
+- Install the dependencies (`pip install -r pip-req.txt`);
+- Install the `nltk` library in development mode
+  (`python setup.py develop`);
+- Download the datasets for running tests
+  (`python -c "import nltk; nltk.download('tests')"`);
```

Comment:
I've created the label `'tests'` to cover all and just the resources needed for running tests. The list of these resources are defined in the `downloader.py` module.

Reply:
Update: Instead of pinning down the list of "tests" packages in `downloader.py`, the list is now defined in the `nltk_data` repo. See https://github.com/nltk/nltk_data/pull/93

Reply:
Use `python -m nltk.downloader tests` instead of `python -c ...` =)